pyqt-apps/siriushla/as_ap_measure/emittance_meas.py

```diff
@@ -404,8 +411,8 @@ def _setupUi(self):
         gl2.addWidget(QLabel('Vert.', resultsgb), 1, 2)
         gl2.addWidget(QLabel('Hor.', resultsgb), 1, 3)
         gl2.addWidget(QLabel('Vert.', resultsgb), 1, 4)
-        gl2.addWidget(QLabel('Norm. emitt. [mm.mrad]', resultsgb), 2, 0)
-        gl2.addWidget(QLabel('beta [m]', resultsgb), 3, 0)
+        gl2.addWidget(QLabel('Norm. emitt.\n[mm.mrad]', resultsgb), 2, 0)
+        gl2.addWidget(QLabel('beta\n[m]', resultsgb), 3, 0)
```

Comment:
@anacso17, I don't think this line break is needed here, the whole beta label is already smaller than the first line of the emittance label. What do you think?

Reply:
@fernandohds564 my idea was to standardize, placing units in the second line. But I think it looks more pleasant without the line break, anyway!

Reply:
Ok, I'll change that then, ok?

Reply:
done!